src/app/shared/services/chirpstack-gateway.service.ts

```diff
@@ -1,87 +1,94 @@
 import { Injectable } from '@angular/core';
 import { RestService } from './rest.service';
 import { Observable } from 'rxjs';
-import { GatewayResponse, Gateway, GatewayData, GatewayRequest, GatewayResponseMany } from '@app/gateway/gateway.model';
+import {
+    GatewayResponse,
+    Gateway,
+    GatewayData,
+    GatewayRequest,
+    GatewayResponseMany,
+} from '@app/gateway/gateway.model';
 import moment from 'moment';
 import { SharedVariableService } from '@shared/shared-variable/shared-variable.service';
 import { map } from 'rxjs/operators';
 import { UserMinimalService } from '@app/admin/users/user-minimal.service';
+import { convertToDateFromTimestamp } from '@shared/helpers/time.helper';
 
 @Injectable({
-  providedIn: 'root'
+    providedIn: 'root',
 })
 export class ChirpstackGatewayService {
+    private chripstackGatewayUrl = 'chirpstack/gateway';
 
-  private chripstackGatewayUrl = 'chirpstack/gateway';
-
-  constructor(
-    private restService: RestService,
-    private sharedVariableService: SharedVariableService,
-    private userMinimalService: UserMinimalService) {
-    moment.locale('da');
-  }
+    constructor(
+        private restService: RestService,
+        private sharedVariableService: SharedVariableService,
+        private userMinimalService: UserMinimalService
+    ) {
+        moment.locale('da');
+    }
 
-  public get(id: string, params = {}): Observable<GatewayResponse> {
-    return this.restService.get(this.chripstackGatewayUrl, params, id)
-      .pipe(
-        map(
-          (response: GatewayResponse) => {
-            response.gateway.internalOrganizationName = this.sharedVariableService.getOrganizationInfo()
-              .find(org => org.id === response.gateway.internalOrganizationId)?.name;
-            // Move createdat and updatedat to next level ease the use.
-            response.gateway.updatedAt = response.updatedAt;
-            response.gateway.createdAt = response.createdAt;
-            response.gateway.lastSeenAt = response.lastSeenAt;
-            response.gateway.createdByName = this.userMinimalService.getUserNameFrom(response.gateway.createdBy);
-            response.gateway.updatedByName = this.userMinimalService.getUserNameFrom(response.gateway.updatedBy);
-            return response;
-          }
-        )
-      );
-  }
+    public get(id: string, params = {}): Observable<GatewayResponse> {
+        return this.restService.get(this.chripstackGatewayUrl, params, id).pipe(
+            map((response: GatewayResponse) => {
+                response.gateway.internalOrganizationName = this.sharedVariableService
+                    .getOrganizationInfo()
+                    .find((org) => org.id === response.gateway.internalOrganizationId)?.name;
+                // Move createdat and updatedat to next level ease the use.
+                response.gateway.tagsString = JSON.stringify(response.gateway.tags)
```

Comment:
Does created/updated at still display properly now that it is not moved here?

Reply:
Yes, it works :-)